docs/content/2.deploy/providers/alwaysdata.md

```diff
@@ -0,0 +1,36 @@
+# alwaysdata
+
+Deploy Nitro apps to alwaysdata.
+
+**Preset:** `alwaysdata` ([switch to this preset](/deploy/#changing-the-deployment-preset))
+
+Nitro supports deploying on the [alwaysdata](https://www.alwaysdata.com/en/) platform with minimal configuration.
+
+## Set up application
+
+### Pre-requisites
+
+1. [Register a new profile](https://www.alwaysdata.com/en/register/) on alwaysdata platform if you don't have one.
+
+2. Get a free 100Mb plan to host your app.
+
+### Local deployment
+
+1. Build your project locally with `NITRO_PRESET=alwaysdata yarn build`
+
+2. [Upload your app](https://help.alwaysdata.com/en/remote-access/) to your account in its own directory (e.g. `$HOME/www/my-app`).
+
+3. On your admin panel, [create a new site](https://admin.alwaysdata.com/site/add/) for your app with the following features:
+   - *Addresses*: `[account-name].alwaysdata.net`
```

Comment:
It seems starting free plan setup, i need to update only account and cannot create new domain like `nitro.alwaysdata.net` it gives "domain does not belongs to you" error.

Should it be "edit exciting domain" ?

Reply:
Right. It's not limited to the free tier: all plans at alwaysdata come with a default URL in the form of `[account_name].alwaysdata.net`. But you can create as many accounts as you want, and you can also link your own domains (or register new ones). So unless your account name is `nitro`, you can't use it as your URL.
I've added a section in the `Create account` pre-requisites to make it clearer. Does it sound good?